### DISCUSSION TITLE
Error using beta repository

### DISCUSSION THREAD

**Discussion by mmol67** [Xpra-org/xpra#3951] [Q&A] [answered]
Hello. Fedora 38 here, using beta repositories. Trying `xpra start-desktop --start=xfce4-session --resize-display=1900x1020`, got 
```
File "/usr/bin/xpra", line 5, in <module>
from xpra.platform import init, set_default_name
MuduleNotFoundError: No module named 'xpra'
```
Xpra from the stable repository is working ok.

**Comment by totaam** [comment, marked as answer]
@friki67 sorry about that.
The workaround is to invoke `python3 /usr/bin/xpra ..`

I will be posted new fixed builds ASAP, the problem comes from #3945.

**Reply by mmol67** [reply]
it will be `python3 /usr/bin/xpra`, and it works! thank you

**Reply by totaam** [reply]
Thanks! That's correct, I've edited my answer.

**Comment by totaam** [comment]
The latest builds do not have this issue.
